frigate/output.py

```diff
@@ -579,9 +582,29 @@ def update(self, camera, object_count, motion_count, frame_time, frame) -> bool:
         if not camera_config.enabled:
             return False
 
+        # get our metrics (sync'd across processes)
+        # which allows us to control it via mqtt (or any other dispatcher)
+        camera_metrics = self.camera_metrics[camera]
+
+        # disabling birdseye is a little tricky
+        if not camera_metrics["birdseye_enabled"].value:
+            # if we've rendered a frame (we have a value for last_active_frame)
+            # then we need to set it to zero
+            if self.cameras[camera]["last_active_frame"] > 0:
+                self.cameras[camera]["last_active_frame"] = 0
+
+            return False
+
+        # default the mode to the camera config at load time
+        # if we have a metric (which we always should as it is set onload)
+        # then use that value
+        birdseye_mode = camera_config.mode
```

Comment:
the metrics have a default value as the config so there is no need to reference the config at all

Reply:
Resolved